### ISSUE TITLE
How to inverse transform the regression predicted labels

### ISSUE THREAD

**Issue by krati-saxena-turing** [georgian-io/Multimodal-Toolkit#25]
Hi,
I have tried using numerical_transformer_method (str, optional) : box_cox and quantile_normal. 
I get the predictions in a different scale as the original.

How to inverse transform the predictions to original labels?

**Comment by akashsaravanan-georgian** [comment]
All numerical transformer methods use scikit-learn under the hood. You should be able to inverse transform by running the `inverse_transform` function. Refer the [Sklearn documentation](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.PowerTransformer.html#sklearn.preprocessing.PowerTransformer) for more information.

Note that you'll likely have to create another instance of the transformer since iirc it isn't saved anywhere when loading the data.
